packages/async-flow/src/replay-membrane.js

```diff
@@ -106,6 +141,25 @@ export const makeReplayMembraneForTesting = ({
       Fail`doReject should only be called on a registered unresolved promise`;
     }
     const guestReason = hostToGuest(hostReason);
+    if (guestReason instanceof Error) {
+      annotateError(
+        guestReason,
+        X(...inlineTemplateArgs`from flow ${flowDescription}`),
+      );
+      if (hostVowToCall) {
+        const hostVowCap = toPassableCap(hostVow);
+        if (hostVowToCall.has(hostVowCap)) {
+          const hostCall = hostVowToCall.get(hostVowCap);
```

Comment:
Is there a reason to leave the host call in the map after rejection? I think doFulfill should remove from the map as well.